src/Console/InstallCommand.php

```diff
@@ -73,8 +73,10 @@ protected function installTests()
 
         $stubStack = $this->argument('stack') === 'api' ? 'api' : 'default';
 
-        if ($this->option('pest')) {
-            $this->removeComposerPackages(['phpunit/phpunit'], true);
+        if ($this->option('pest') || $this->isUsingPest()) {
+            if ($this->hasComposerPackage('phpunit/phpunit')) {
+                $this->removeComposerPackages(['phpunit/phpunit'], true);
+            }
```

Comment:
This solves an existing minor issue when the user already has Pest. Previously the console output would show a warning about the package not existing, and it would unnecessarily run the composer scripts, which takes a bit of time.